### ISSUE TITLE
verify node.js version before starting ucoind

### ISSUE THREAD

**Issue by Sylvain303** [ucoin-io/ucoin#252]
Example of fail start, after a VM reboot. (I forgot to switch to the correct node.js version)

Useful for non friendly node.js  peoples:

~~~
/home/ucoin/ucoinsrc/bin/ucoind:129
        return server.disconnect().catch(() => null);
                                          ^
SyntaxError: Unexpected token )
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
ucoin@debian-jessie:~/ucoinsrc$ node --version
v0.10.29
~~~

Fix, for me:

~~~
nvm use 0.12
Now using node v0.12.9 (npm v2.14.9)
~~~

starting OK…

~~~
node --harmony bin/ucoind start
~~~

**Comment by c-geek** [comment]
Well the official starting point is [ucoin.sh](https://github.com/ucoin-io/ucoin/blob/master/ucoin.sh), and it takes care of this.

But if you want to add the line testing for the good Node.js version in [ucoind](https://github.com/ucoin-io/ucoin/blob/master/bin/ucoind), do not hesitate :-)

**Comment by c-geek** [comment]
I let you eventually fix this, but I prefer to close the issue and set it as duplicate of #166 which will make use of `--harmony` option completely useless.
